webpack.config.js

```diff
@@ -1,13 +1,37 @@
 const autoprefixer = require('autoprefixer');
 const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const getConfig = require('./get-config');
 const spawnSync = require('child_process').spawnSync;
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const MonacoWebpackPlugin = require('monaco-editor-webpack-plugin');
 const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
+const fs = require('fs');
 
-// Object.assign(global, { WebSocket: require('ws') });
+function readConfig() {
+  const configPath = fs.existsSync('./config-local.json')
+    ? './config-local.json'
+    : './config.json';
```

Comment:
Is it worth checking for `config.json` existence here and fail if it's not available? Or... can we trust the code later on to fail appropriately if this happens?

Reply:
> Or... can we trust the code later on to fail appropriately if this happens?

I should have kept scrolling... There is a clear `console.error` below that handles the case.